src/utils/navigation/build_http_url.js

```diff
@@ -0,0 +1,6 @@
+export default function buildHttpUrl() {
+  const isSSL = process.env.REACT_APP_DEVELOPMENT === "true";
```

Comment:
There is a built-in environment variable called `NODE_ENV.` You can read it from `process.env.NODE_ENV`. We can use it for this purpose. It could be `development ` or `production`

Reply:
When you run npm start, it is always equal to 'development', when you run npm test it is always equal to 'test', and when you run npm run build to make a production bundle, it is always equal to 'production'.


taken from here https://create-react-app.dev/docs/adding-custom-environment-variables/